test/service/randoServiceTest.js

```diff
@@ -12,6 +12,38 @@ var Errors = require("../../src/error/errors");
 describe('Rando service.', function () {
     describe('Save image.', function () {
 
+	 	//restore all stubs because we can have a case:
+	 	//Call parralel two funtions and both use stubs. One of stub can be frozen and break next test.
+    	afterEach(function() {
```

Comment:
> Issue found: Function 'anonymous' has a complexity of 8. (complexity)

Ignore this suggestion, because `clear stubs` code in the test must be super easy to read  and extremely stupid because we want have a maintainable code in the future.